website/snippets/_adapters-trusted.md

```diff
@@ -2,7 +2,10 @@
 
 <Card
     title="Athena"
-    body="<ul><li><a href='/docs/core/connect-data-platform/athena-setup'>Install with dbt Core </a> </li> </ul><br /><br /><a href=https://badge.fury.io/py/dbt-athena-community><img src=https://badge.fury.io/py/dbt-athena-community.svg/></a>"
+    body="<ul><li><a href='/docs/core/connect-data-platform/athena-setup'>Install with dbt Core </a> </li> </ul><br /><br /><a href=https://badge.fury.io/py/dbt-athena-community><img src=https://badge.fury.io/py/dbt-athena-community.svg/></a><br /><br /><div style='background: #E5E7EB;
+  border-radius: 1.5rem;
+  color:#262A38;
+  padding: 0rem 0.75rem; display:inline-block'>dbt Core</div>"
```

Comment:
@nghi-ly is there a way to add multiple pills? So we would need Athena to have `dbt core` and `Trusted Adapter`. I can't tell if this is the code needed `display:inline-block'>dbt Core</div>"`

Reply:
👍 ✅ 

played around with adding the string "Trusted Adapter". easy to change it. please see slack thread for extra context.